vendor/bat-native-ads/src/bat/ads/internal/account/confirmations/confirmations_state.cc

```diff
@@ -341,8 +341,8 @@ base::Value ConfirmationsState::GetFailedConfirmationsAsDictionary(
     if (user_data && user_data->is_dict()) {
       base::DictionaryValue* user_data_dictionary = nullptr;
       if (user_data->GetAsDictionary(&user_data_dictionary)) {
-        confirmation_dictionary.SetKey("user_data",
-                                       base::Value(user_data_dictionary));
+        confirmation_dictionary.SetKey(
+            "user_data", base::Value(std::move(*user_data_dictionary)));
```

Comment:
`user_data_dictionary->Clone` is what you want here instead of `base::Value(std::move(*user_data_dictionary))`

Reply:
using `std::move` here will also affect `user_data`. Although it isn't used again here, that isn't intuitive and clone is preferred anyway

Reply:
Addressed in 23679ec4f8c14655b290449de52d8cd4680e70a7. Also in there removed a bunch of base::Value wrapper objects that I don't think are necessary.